### ISSUE TITLE
The download URL may be disabled

### ISSUE THREAD

**Issue by bioinformaticspcj** [gjospin/PhyloSift#492]
Hi, I just try to download the phylosift, but I can not reach the URL :"http://edhar.genomecenter.ucdavis.edu/~koadman/phylosift/phylosift_latest.tar.bz2"
was it disabled?

**Comment by gjospin** [comment]
Yes it was. Please use the following link to manually install the database. 

https://figshare.com/articles/PhyloSift_markers_database/5755404/4

**Comment by manli-zou** [comment]
There is a problem with this link, it is more like a web page instead of a database 
https://figshare.com/articles/PhyloSift_markers_database/5755404/4

**Comment by gjospin** [comment]
Yes and from there, there are download links that will let you download the
data files.

**Comment by manli-zou** [comment]
Thanks.     
but no matter which link I clicked in , another web page showed out. 
Could you nicely help me check it

**Comment by gjospin** [comment]
Strange.
Let me see if I can describe what I do.

So at the top of the page, I see 4 boxes, each representing 1 tarball or
file to download. If you mouseover the boxes the word "view" and an arrow
will appear at the bottom of the box.
Can you try click on the arrow and see if that downloads the file for you?
You will need ncbi.tgz and markers.tgz

**Comment by manli-zou** [comment]
The problem is solved. 
Thank you so much ~

Things went all well when I turned to use Microsoft edge instead of Google Chrome.
Maybe this is the reason.

**Comment by osvatic** [comment]
The link provided is now broken. Can you please provide a new link for the downloading of phylosift dbs?

**Comment by gjospin** [comment]
Which link are you referring to?
The following seems to be working fine this morning. This URL will not work
within the automatic download code. You will need to manually untar and
place the DB in the right directory.
https://figshare.com/articles/PhyloSift_markers_database/5755404/4
